### DISCUSSION TITLE
Deploying to Google App Engine,

### DISCUSSION THREAD

**Discussion by falcorp-cloud** [directus/directus#14575] [Archive]
Previous version of Directus I used v9.8.0 worked fine under google app engine current version fails with endpoint /serve not found. Anyone managed to deploy latest Directus to App Engine?

**Comment by rijkvanzanten** [comment]
There has never been a `/serve` endpoint, so I'm not sure what to make of that 🤔

**Reply by falcorp-cloud** [reply]
App Engine by default results to /serve as entry point. Generally the package.json "start" entry would resolve the issue however fresh deploy of current version produces error "Unable to find entrypoint /serve", this would mean that "start" script is somewhat ignored.

**Reply by falcorp-cloud** [reply]
versions passed v9.8.0 don't work, seems like .env super seeds app.yaml/app engine environment configurations. I got v9.14.5 working by updating port to 8081 in .env other container provided configurations are ignored. I remember this issue existed for one of the earlier RC versions.

**Reply by falcorp-cloud** [reply]
I have triple checked and confirmed it, any env configurations we make are ignored. This issue https://github.com/directus/directus/issues/4191 is back

**Reply by azrikahar** [reply]
Ref #14587 in case we'd like to consolidate the same discussion in one place. I assume this was opened by or on behalf of OP, but could just be a coincidence! 🙂
